examples/client.c

```diff
@@ -5,12 +5,12 @@
 # include "networklayer_tcp.h"
 # include "logger_stdout.h"
 # include "ua_types_encoding_binary.h"
+#include "server/ua_nodes.h"
```

Comment:
Including an internal API from the server for the client...
There should be a better way.